### ISSUE TITLE
$user->newSubscription('main' $plan)->create(); Fails

### ISSUE THREAD

**Issue by bezhanSalleh** [oanhnn/laravel-pricing-plans#3]
$user->newSubscription('main',$plan)->create();
Illuminate/Database/QueryException with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'plans.plan_id' in 'where clause' (SQL: select * from `plans` where `plans`.`plan_id` is null limit 1)'

**Comment by oanhnn** [comment]
Can you show more info?
`User` model ?

**Comment by bezhanSalleh** [comment]
i exactly followed the steps as you described in the package intro doc, nothing more.

**Comment by Georde** [comment]
Illuminate/Database/QueryException with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'plans.plan_id' in 'where clause' (SQL: select * from plans where plans.plan_id is null limit 1)'

**Comment by Georde** [comment]
Before
```php
public function plan()
{
    return $this->belongsTo(
        Config::get('plans.models.Plan'),
        Config::get('plans.tables.plans'),
        'plan_id',
        'id'
    );
}
```
after
```php
public function plan()
{
    return $this->belongsTo(
        Config::get('plans.models.Plan')
    );
}
```
 in `BelongsToPlanModel.php`
OK

**Comment by oanhnn** [comment]
Thank @bezhanSalleh @Georde 
I fixed in v0.3.2
